tests/src/common/test_case/builder.rs

```diff
@@ -200,14 +202,14 @@ where
 {
     pub fn init_profit(
         self,
-        cadence_hours: u16,
+        cadence_hours: CadenceHours,
```

Comment:
I found other occurrences of `u16` instead of `CadenceHours` in the tests, please replace all.